internal/statestore/redis.go

```diff
@@ -192,7 +183,7 @@ func (rb *redisBackend) CreateTicket(ctx context.Context, ticket *pb.Ticket) err
 	if rb.cfg.IsSet("redis.expiration") {
 		redisTTL := rb.cfg.GetInt("redis.expiration")
 		if redisTTL > 0 {
-			err = redisConn.Send("EXPIRE", ticket.GetId(), redisTTL)
+			_, err = redisConn.Do("EXPIRE", ticket.GetId(), redisTTL)
```

Comment:
revert any change here as this is redundant with https://github.com/googleforgames/open-match/pull/1146?